nvfuser/__init__.py

```diff
@@ -58,7 +58,25 @@ def __enter__(self):
         return self._setup_definition()
 
     def __exit__(self, type, value, traceback):
-        self._finalize_definition()
+        try:
+            self._finalize_definition()
+        except Exception as err:
+            msg = (
```

Comment:
nitpick, can we have this part in a separate question and reuse it with the section in execute? just so that we don't have duplicated strings. :stuck_out_tongue_winking_eye: